azure-functions/acbs-function/src/functions/acbs-amend-facility-covenant-record.js

```diff
@@ -24,9 +24,26 @@
  * This function is responsible for amending Facility Covenant Record.
  */
 const df = require('durable-functions');
-const retryOptions = require('../../helpers/retryOptions');
+const retry = require('../../helpers/retry');
 const mappings = require('../../mappings');
 
+/**
+ * Durable Orchestration Function (DOF) for amending a facility covenant record in the ACBS system.
+ *
+ * This function is triggered by an HTTP trigger function and performs the following operations:
+ * 1. Validates the input payload.
+ * 2. Maps the amendments to the facility covenant record.
+ * 3. Updates the facility covenant record in the ACBS system.
+ *
+ * @param {Object} context - The context object provided by Durable Functions.
```

Comment:
Might be worth doing a find replace for `{Object}` to `{object}` :)

Reply:
Merged under PR #3600